auto_py_to_exe/web/js/i18n.js

```diff
@@ -993,7 +1003,7 @@ const translationMap = {
                 zh_tw: '如何產生',
                 ua: 'Налаштування генератора',
                 ru: 'Настройки генератора',
-                de: 'Wie generieren...?',
+                de: 'Wie generieren?',

```

Comment:
@ackhh not sure if we need to keep the "?" here? It's just a heading, not a question.

Reply:
You are right, "?" can be omitted